src/NuGet.Core/NuGet.Packaging/Signing/TrustStore/FallbackCertificateBundleX509ChainFactory.cs

```diff
@@ -14,7 +14,8 @@ internal sealed class FallbackCertificateBundleX509ChainFactory : CertificateBun
     {
         // These constants are dictated by the .NET SDK.
         internal const string SubdirectoryName = "trustedroots";
-        internal const string FileName = "codesignctl.pem";
+        internal const string CodeSigningFileName = "codesignctl.pem";
```

Comment:
Can you please update our `.gitattributes` to treat `.pem` files as binary so nothing tries to display a diff for them?